test/requests/completion_resolve_test.rb

```diff
@@ -0,0 +1,39 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+
+class CompletionResolveTest < Minitest::Test
+  include RubyLsp::Requests::Support::Common
+
+  Interface = LanguageServer::Protocol::Interface
+  Constant = LanguageServer::Protocol::Constant
+
+  def test_completion_resolve_for_constant
+    stub_no_typechecker
+    source = +<<~RUBY
+      class Foo
```

Comment:
Let's add a couple of comments here to ensure there's some documentation being returned.